pkg/docker/cockpit-atomic-storage

```diff
@@ -0,0 +1,482 @@
+#! /usr/bin/python
```

Comment:
Is this python2 or python3? This is the first part of the Cockpit runtime where we are running python on the target system. We should probably make this script compatible with both, and run it from the browser, rather than install it.